Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py

```diff
@@ -764,6 +875,117 @@ def fetch_incidents(client, aws_sh_severity, archive_findings, additional_filter
+    demisto.debug(f'Performing get-remote-data command with incident id: {remote_incident_id} and {last_update_time=}')
+
+    filters = {
+        'Id': [
+            {
+                'Comparison': 'EQUALS',
+                'Value': remote_incident_id
+            }
+        ]
+    }
+    response = client.get_findings(Filters=filters)
+    demisto.debug(f'The response is: {response} \nEnd of response.')
+    finding = response.get('Findings')[0]  # a list with one dict in it
+    incident_last_update = finding.get('UpdatedAt', '')
+    incident_last_update_time = last_update_to_time(incident_last_update)
+    demisto.debug(f'The incident last update time is: {incident_last_update}\nAnd {incident_last_update_time=}')
+
+    if last_update_time < incident_last_update_time:
+        demisto.debug(f'Updated incident {remote_incident_id}')
+        return GetRemoteDataResponse(mirrored_object=finding, entries=[{}])
+    else:
+        demisto.debug('Nothing new in the incident.')
+        return GetRemoteDataResponse(mirrored_object={}, entries=[{}])
+
+
+def get_mapping_fields_command() -> GetMappingFieldsResponse:
+    """
+    Returns the list of fields to map in outgoing mirroring, for incidents and detections.
+    """
+    incident_type_scheme = SchemeTypeMapping(type_name='AWS Security Hub Finding')
+    demisto.debug('Collecting incident mapping.')
+
+    for field in OUT_FIELDS + FindingIdentifiers_lIST:
+        incident_type_scheme.add_field(field)
+
+    mapping_response = GetMappingFieldsResponse()
+    mapping_response.add_scheme_type(incident_type_scheme)
+
+    return mapping_response
+
+
+def update_remote_system_command(client: boto3.client, args: Dict[str, Any], closure_case: str) -> str:
+    """
+    Mirrors out local changes to the remote system.
+    Args:
+        client: boto3.client - AWS client
+        args: A dictionary containing the data regarding a modified incident, including: data, entries,
+            incident_changed, remote_incident_id, inc_status, delta.
+        closure_case: str - Whether to resolve an incident in Security Hub, that was closed in XSOAR.
+
+    Returns:
+        The remote incident id that was modified. This is important when the incident is newly created remotely.
+    """
+    parsed_args = UpdateRemoteSystemArgs(args)
+    delta = parsed_args.delta
+    remote_incident_id = parsed_args.remote_incident_id
+    data = parsed_args.data
+    list_delta_keys = []
+    demisto.debug(f'Got the following {parsed_args.inc_status=}, {data=}, {delta=} '
+                  f'and remote ID: {remote_incident_id}.')
+    if delta:
+        list_delta_keys = list(delta.keys())
+        demisto.debug(f'Got the following delta keys {list(delta.keys())}.')
+    if parsed_args.incident_changed:
+        demisto.debug(f'Incident id {remote_incident_id} and incident change: {parsed_args.incident_changed}')
+        if parsed_args.inc_status == IncidentStatus.DONE and closure_case == 'Resolved':
```

Comment:
the new param should be a checkbox and not a string,
Just if to close it or not